### ISSUE TITLE
Support logical date and timestamp types

### ISSUE THREAD

**Issue by hsn10** [lerouxrgd/rsgen-avro#51]
I need support for:

    {"name":"day", "type" : {"type":"int", "logicalType":"date"}},

and

    {"name":"start",            "type" : {"type":"long", "logicalType":"timestamp-millis"}},

**Comment by hsn10** [comment]
Logical types are supported in Official Rust version of AVRO:

https://github.com/apache/avro/blob/master/lang/rust/avro/src/schema.rs

I don't really understand their official rust code.
Do they deserialize logical types to integer/long? In this case, their support will be just adding _logicalType_ label to reading/writing schema. In Java these types are deserialized to java.time.LocalTime;

**Comment by lerouxrgd** [comment]
I made a basic implementation on [this branch](https://github.com/lerouxrgd/rsgen-avro/tree/chrono-dates) with [chrono](https://crates.io/crates/chrono) (You then need to add it to your `Cargo.toml` with the `serde` feature).

Could you give it a try with:

```
git clone https://github.com/lerouxrgd/rsgen-avro.git
cd rsgen-avro
cargo run -- <path/to/your/schema.avsc> --chrono-dates -
```

Let me know how it goes.

**Comment by lerouxrgd** [comment]
Released in version `0.11.11`, don't hesitate to re-open if something is wrong.

**Comment by hsn10** [comment]
I need to test it against spark. if timezone is correct.
